src/Libraries/Microsoft.Extensions.DataIngestion.Abstractions/IngestionChunk.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.DataIngestion;
+
+/// <summary>
+/// Represents a chunk of content extracted from an <see cref="IngestionDocument"/>.
+/// </summary>
+/// <typeparam name="T">The type of the content.</typeparam>
+[DebuggerDisplay("{Content}")]
```

Comment:
Nit: typically you'd have `"Content = {Content}"`. Is the debugging experience better here without that?